### PR TITLE
add ez_api.1.0.0

### PR DESCRIPTION
**Pull request by RadioPotin**
Hi,

Here's the latest `ez_api` version to date, fixed yesterdays' mishaps ^^

Until next time

### CODE REVIEW

File: packages/ez_api/ez_api.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  "digestif" {>= "1.0.0"}
+]
+conflicts: [
+  "calendar" {< "2.03"}
+  "cohttp-lwt-jsoo" {< "4.0.0"}
+  "httpaf-lwt-unix" {< "0.6.0"}
+  "ezjs_fetch" {< "0.2"}
+  "ocurl" {< "0.8.0"}
+  "digestif" {< "1.0.0"}
+]
```
Comment (mseri) [inline packages/ez_api/ez_api.1.0.0/opam:47]:
```suggestion
  "result" {< "1.5"}
]
```

File: packages/ez_api/ez_api.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ocamlpro/ez_api"
+doc: "https://ocpmax.github.io/ez_api/ez_api/index.html"
+bug-reports: "https://github.com/ocamlpro/ez_api/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.08"}
+  "json-data-encoding" {>= "0.7.0"}
+  "lwt" {>= "4.0.0"}
+  "ezjsonm" {>= "1.0.0"}
```
Comment (mseri) [inline packages/ez_api/ez_api.1.0.0/opam:20]:
```suggestion
  "ezjsonm" {>= "1.1.0"}
```

Uses Ezjsonm.values_from_string

File: packages/ez_api/ez_api.1.0.0/opam [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  "Alain Mebsout <alain.mebsout@ocamlpro.com>"
+]
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ocamlpro/ez_api"
+doc: "https://ocpmax.github.io/ez_api/ez_api/index.html"
+bug-reports: "https://github.com/ocamlpro/ez_api/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.08"}
+  "json-data-encoding" {>= "0.7.0"}
```
Comment (mseri) [inline packages/ez_api/ez_api.1.0.0/opam:18]:
```suggestion
  "json-data-encoding" {>= "0.9"}
```
Due to `String {string_specs with str_format = Some "binary"} )`

**Comment by mseri** [comment]
There is a failure on 4.10 that I don't know how to address:
```
# (cd _build/default && /home/opam/.opam/4.10/bin/ocamlc.opt -w -40 -w +a-4-41-44-45-48 -g -bin-annot -I src/ppx/.ppx_common.objs/byte -I /home/opam/.opam/4.10/lib/base -I /home/opam/.opam/4.10/lib/base/base_internalhash_types -I /home/opam/.opam/4.10/lib/base/caml -I /home/opam/.opam/4.10/lib/base/shadow_stdlib -I /home/opam/.opam/4.10/lib/ocaml-compiler-libs/common -I /home/opam/.opam/4.10/lib/ocaml-compiler-libs/shadow -I /home/opam/.opam/4.10/lib/ocaml-migrate-parsetree -I /home/opam/.opam/4.10/lib/ocaml/compiler-libs -I /home/opam/.opam/4.10/lib/ppx_derivers -I /home/opam/.opam/4.10/lib/ppxlib -I /home/opam/.opam/4.10/lib/ppxlib/ast -I /home/opam/.opam/4.10/lib/ppxlib/print_diff -I /home/opam/.opam/4.10/lib/ppxlib/traverse_builtins -I /home/opam/.opam/4.10/lib/result -I /home/opam/.opam/4.10/lib/sexplib0 -I /home/opam/.opam/4.10/lib/stdio -I src/ppx/compat/.ppx_compat.objs/byte -I src/ppx/compat/.ppx_compat.objs/native -no-alias-deps -o src/ppx/.ppx_common.objs/byte/ppx_common.cmo -c -impl src/ppx/ppx_common.ml)
# File "src/ppx/ppx_common.ml", line 312, characters 6-25:
# 312 |     | Pmod_functor (f, m) -> {pmod with pmod_desc = Pmod_functor (f, pmod_impl m)}
#             ^^^^^^^^^^^^^^^^^^^
# Error: The constructor Pmod_functor expects 3 argument(s),
#        but is applied here to 2 argument(s)
```

**Comment by RadioPotin** [comment]
Me neither. I will refer it to the relevant people and come back to you with a solution

**Comment by mseri** [comment]
Thanks!

**Pull request merged**